### ISSUE TITLE
Fix extension versioning consistency

### ISSUE THREAD

**Issue by keithf4** [theory/pgtap#94]
The pgtap.control file has the version at 0.95.0
The sql folder has a file called pgtap--0.95.0--0.96.0.sql
The META.json and doc files have the version 0.95.1 which was never an actually released version that I can see

This is really confusing what the currently supported version is supposed to be

**Comment by theory** [comment]
The current release is 0.95.0. The current dev version is 0.95.1. The next expected release will be v0.96.0. Probably best to rely on versions in the [released version](http://pgxn.org/dist/pgtap/), and just expect versions in the Git repo to be incremented from there as appropriate. The next released zip file will have everything consistently at v0.96.0.

Does that help?

**Comment by keithf4** [comment]
I figured that was the case with the 0.96.0 update file. The 0.95.1 really confused me, though because I only saw it mentioned in docs and no releases involved with it.
Going 5 months with stuff just sitting there with no clear explanation on how to interpret the version status is rather confusing, though. Especially for someone just coming to the project for the first time.

Just trying to help move things along for a tool that I find indispensable myself :)
